doc/images/Hello.md

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
Seems this file is not needed.

Reply:
Yeah. Github didn't allow me to create empty folder and hence I created this. Will remove it now.

Reply:
@swatid-seagate - Now it's showing two folders - Docs - https://github.com/swatid-seagate/cortx-re/tree/sodapage-upload/docs/images and https://github.com/swatid-seagate/cortx-re/tree/sodapage-upload/doc 

You can clone your branch on linux box and just to `mv doc docs` then commit change and push. This will fix isssue.